packages/commands/src/create-playlist.mjs

```diff
@@ -46,6 +64,8 @@ program
             description || `playlist generated by audius-cmd ${rand}`,
           is_album: album,
           is_private: isPrivate,
+          release_date: releaseDate,
+          is_scheduled_release: isScheduledRelease,
```

Comment:
ideally we would set this automatically by comparing dates instead of having user pass it in manually, but didn't want to install another package just for this.